Analysis/Tasks/PWGHF/CMakeLists.txt

```diff
@@ -23,11 +23,21 @@ o2_add_dpl_workflow(hf-candidate-creator-2prong
                     PUBLIC_LINK_LIBRARIES O2::Framework O2::AnalysisDataModel O2::AnalysisCore O2::DetectorsVertexing ROOT::EG
                     COMPONENT_NAME Analysis)
 
+o2_add_dpl_workflow(hf-tree-creator-d0-to-k-pi
```

Comment:
Let's not overhyphen it. :-) `hf-tree-creator-d0-tokpi` looks explicit enough to me.
We already have `hf-jpsi-toee-candidate-selector`, so better to stay consistent if possible.